src/test/java/nl/dtls/fairdatapoint/api/controller/MetadataControllerTest.java

```diff
@@ -90,7 +90,7 @@
     private final String TEST_DISTRIBUTION_PATH = TEST_FDP_PATH
             + "/distribution/" + ExampleFilesUtils.DISTRIBUTION_ID;
     private final static Logger LOGGER
-            = LogManager.getLogger(FairMetaDataServiceImpl.class.getName());
+            = LoggerFactory.getLogger(FairMetaDataServiceImpl.class.getName());
```

Comment:
We can use the class itself instead of resolving it per name. (also for consistency)